src/custom/components/swap/EthWethWrap/helpers.ts

```diff
@@ -35,11 +38,29 @@ export const _getAvailableTransactions = ({
 }: {
   nativeBalance?: CurrencyAmount<Currency>
   nativeInput?: CurrencyAmount<Currency>
-  singleTxCost: CurrencyAmount<Currency>
+  singleTxCost?: CurrencyAmount<Currency>
 }) => {
-  if (!nativeBalance || !nativeInput || nativeBalance.lessThan(nativeInput.add(singleTxCost))) return null
+  if (!nativeBalance || !nativeInput || !singleTxCost || nativeBalance.lessThan(nativeInput.add(singleTxCost))) {
+    return null
+  }
 
   // USER_BALANCE - (USER_WRAP_AMT + 1_TX_CST) / 1_TX_COST = AVAILABLE_TXS
   const txsAvailable = nativeBalance.subtract(nativeInput.add(singleTxCost)).divide(singleTxCost)
   return txsAvailable.lessThan('1') ? null : txsAvailable.toSignificant(1)
 }
+
+export function _estimateTxCost(gasPrice: ReturnType<typeof useGasPrices>, native: Currency | undefined) {
+  if (!native) {
+    return {}
+  }
+  // TODO: should use DEFAULT_GAS_FEE from backup source
+  // when/if implemented
+  const gas = gasPrice?.standard || DEFAULT_GAS_FEE
+
+  const amount = BigNumber.from(gas).mul(MINIMUM_TXS).mul(AVG_APPROVE_COST_GWEI)
+
+  return {
+    multiTxCost: CurrencyAmount.fromRawAmount(native, amount.toString()),
```

Comment:
nit: I felt the name was a bit hard to understand. What about `recomendedMarginalEther`? i also don't love it. Maybe you find a better one

Reply:
I'm just moving this fn around.
If you want to complain about the names bother @W3stside =P

But ok... I'll refactor this next week when I'm more in the mood.

Reply:
its just a nit, the PR was approved :)

Reply:
i wrote this? 😂